WalletWasabi.Gui/UiConfig.cs

```diff
@@ -47,6 +47,10 @@ public class UiConfig : ConfigBase
 		[JsonProperty(PropertyName = "FeeDisplayFormat", DefaultValueHandling = DefaultValueHandling.Populate)]
 		public int FeeDisplayFormat { get; internal set; }
 
+		[DefaultValue("")]
+		[JsonProperty(PropertyName = "Tab", DefaultValueHandling = DefaultValueHandling.Populate)]
+		public string Tab { get; internal set; }
```

Comment:
`Tab` is no very descriptive. Could you use a better name here? `LastActiveTab`, `ActiveTab` or something like that.

Reply:
done.